### ISSUE TITLE
The incompatibility/bug that never seems to go away.

### ISSUE THREAD

**Issue by circuitikzuser** [DeaDBeeF-Player/deadbeef#2735]
Deadbeef and GTK seems to be worst enemies, and the deadbeef-qt fork seems dead, so no luck there. 
Wherever I read over the years  of this perpetual bug, or perpetual incompatibility between deadbeef and GTK, users solved the issue using deadbeef-QT.
If anyone has a solution for installing deadbeef-qt let me know. 
I really need it as deadbeef just died recently with the following errors.
If you click on a song and press play button deadbeef crashes.
Here is the complete terminal output.
It seems deadbeef loads both gtk2&3 and that can only be trouble in my humble opinion.

```
$deadbeef
starting deadbeef 1.8.4
server_start
searching for GUI plugins in /home/testuser/.local/lib64/deadbeef
searching for GUI plugins in /home/testuser/.local/lib/deadbeef
searching for GUI plugins in /usr/lib/x86_64-linux-gnu/deadbeef
load_plugin_dir /usr/lib/x86_64-linux-gnu/deadbeef: scandir found 106 files
found gui plugin ddb_gui_GTK2.so
added GTK2 gui plugin
found gui plugin ddb_gui_GTK3.so
added GTK3 gui plugin
load gui plugin
checking GUI plugin: GTK2
found selected GUI plugin: GTK2
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/ddb_gui_GTK2.so
loading plugins from /home/testuser/.local/lib64/deadbeef
loading plugins from /home/testuser/.local/lib/deadbeef
loading plugins from /usr/lib/x86_64-linux-gnu/deadbeef
load_plugin_dir /usr/lib/x86_64-linux-gnu/deadbeef: scandir found 106 files
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/aac.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/adplug.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/alac.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/alsa.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/artwork.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/cdda.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/converter.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/converter_gtk2.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/converter_gtk3.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/dca.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/ddb_dumb.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/ddb_mono2stereo.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/ddb_shn.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/ddb_soundtouch.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/dsp_libsrc.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/ffap.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/ffmpeg.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/flac.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/gme.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/hotkeys.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/in_sc68.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/lastfm.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/m3u.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/mms.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/mp3.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/musepack.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/notify.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/nullout.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/opus.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/oss.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/pltbrowser_gtk2.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/pltbrowser_gtk3.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/psf.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/pulse.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/rg_scanner.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/shellexec.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/shellexecui_gtk2.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/shellexecui_gtk3.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/sid.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/sndfile.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/supereq.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/tta.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/vfs_curl.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/vfs_zip.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/vorbis.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/vtx.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/wavpack.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/wildmidi.so
loading plugin /usr/lib/x86_64-linux-gnu/deadbeef/wma.so
starting plugin GTK2 user interface
starting plugin AAC player
starting plugin Adplug player
starting plugin ALAC player
starting plugin ALSA output plugin
starting plugin Album Artwork
starting plugin Audio CD player
starting plugin Converter
starting plugin Converter GTK2 UI
starting plugin Converter GTK3 UI
starting plugin dts decoder
starting plugin DUMB module player
starting plugin Mono to stereo
starting plugin Shorten player
starting plugin Soundtouch
starting plugin Resampler (Secret Rabbit Code)
starting plugin Monkey's Audio (APE) decoder
starting plugin FLAC decoder
starting plugin Game-Music-Emu player
starting plugin Hotkey manager
starting plugin SC68 player (Atari ST SNDH YM2149)
starting plugin last.fm scrobbler
starting plugin M3U and PLS support
starting plugin mms vfs
starting plugin MP3 player
starting plugin MusePack decoder
starting plugin OSD Notify
starting plugin Null output plugin
starting plugin Opus player
starting plugin OSS output plugin
starting plugin Playlist browser GTK2
starting plugin Playlist browser GTK3
starting plugin PSF player using Audio Overload SDK
starting plugin PulseAudio output plugin
starting plugin ReplayGain Scanner
starting plugin Shell commands
starting plugin Shellexec GTK2 UI
starting plugin Shellexec GTK3 UI
starting plugin SID player
starting plugin WAV/PCM player
starting plugin SuperEQ
starting plugin tta decoder
starting plugin cURL vfs
starting plugin ZIP vfs
starting plugin Ogg Vorbis decoder
starting plugin VTX player
starting plugin WavPack decoder
starting plugin WildMidi player
starting plugin WMA player
starting plugin stdio vfs
starting plugin FFMPEG audio player
selected output plugin: PulseAudio output plugin
INFO: from file /home/testuser/.config/deadbeef/playlists/0.dbpl
INFO: from file /home/testuser/.config/deadbeef/playlists/1.dbpl
INFO: from file /home/testuser/.config/deadbeef/playlists/2.dbpl
INFO: from file /home/testuser/.config/deadbeef/playlists/3.dbpl
INFO: from file /home/testuser/.config/deadbeef/playlists/4.dbpl
INFO: from file /home/testuser/.config/deadbeef/playlists/5.dbpl
INFO: from file /home/testuser/.config/deadbeef/playlists/6.dbpl
INFO: from file /home/testuser/.config/deadbeef/playlists/7.dbpl
convgui: gtkui plugin not found
plugin Converter GTK3 UI failed to connect to dependencies, deactivated.
plugin Playlist browser GTK3 failed to connect to dependencies, deactivated.
shellexecui: can't find gtkui plugin
plugin Shellexec GTK3 UI failed to connect to dependencies, deactivated.
resume: track -1 pos -1.000000 playlist -1
gtkui plugin compiled for gtk version: 2.24.33
connecting button tray signals
Segmentation Fault
backtrace() returned 2 addresses
deadbeef(+0x34b8d) [0x55d62e761b8d]
/lib/x86_64-linux-gnu/libc.so.6(+0x3bd60) [0x7f598d40bd60]

```

**Comment by Alexey-Yakovenko** [comment]
@circuitikzuser have you considered to use the latest release, instead of 1.5 year old one? I can't see any indication in the log, that the crash could have been caused by a GTK incompatibility.
Also you didn't provide any information about your system, so nothing I can action upon.

**Comment by circuitikzuser** [comment]
Alexey, I just solved it. It is the weirdes thing. Deadbeef just died for no reason and would crash as soon as there are cdemu images. I tried to use it without the .config/deadbeef/playlists file and start from scratch and let deadbeef configure a new playlist file etc. I did everything from a  blank config/dpkg --purge deadbeef and the result always the same.

This is a common problem and everyone says it is a corrupt playlist. In my case my playlists are backed up so no chance for that and the current playlist is diff identical to the backed up versions.

I found that the MX21 deadbeef package is completely junk for some reason. New distro installed apt deadbeef and it installed but failed upon plaing isos.

the solution I just found now is that ONLY the following page works correctly (but ignores my custom deadbeef gui layout why ?) 
https://deadbeef.sourceforge.io/download.html

The sourceforge.net etc deadbeef packages are seemingly corrupt. All of them. And packagers probably used that instead of.
https://deadbeef.sourceforge.io/download.html
These days it seems wherever gnome and its evil master systemD wanders applications fail.

Must I take the apt failures up with MX, or how does it work.?

**Comment by Alexey-Yakovenko** [comment]
Typically, the official build(s) from the deadbeef download page are more reliable, since they're made with specific versions of libraries, which have been tested and proven to work correctly.

The other unofficial packages that you have tried are probably using whatever versions of the libraries which are available in a specific distribution, and there's no way to test all possible combinations and guarantee that they work.

Although it's a bit confusing when you are saying "packages from download.html" vs "packages from sourceforge.net", which makes it impossible for me to know which packages you name corrupt, because technically ALL official builds of deadbeef are on sourceforge.net.

Therefore can you be more specific, and provide a more concrete examples of corrupt vs non-corrupt files?

**Comment by circuitikzuser** [comment]
As an example this failed horribly from source when none of the deb versions on sourceforge.net wanted to work.. https://sourceforge.net/projects/deadbeef/files/debian/
ANYTHING deadbeef and sourceforge with an orange and gray html scheme.

The deb versions e.g. here are absolutely junk. Usually I get good results here in general.
https://pkgs.org/download/deadbeef

Just after I posted the issue here I found https://deadbeef.sourceforge.io/download.html.
The black background threw me probably earlier and I must have assumed at a glance it is not sourceforge as the color scheme was wrong.
Well it is the ONLY version that will work with MX21 with iso images. Everything else plays on other versions of deadbeef on sourceforge.net etc but NOT playing isos.

I also tried every version down the line for Biullseye buster Sid, and everything down to earliest versions here when the latest failed from source and deb.

"https://sourceforge.net/projects/deadbeef/files/"

the problem is seemingly that the packagers at MX might have used the junk on sourceforge.net and pkgs.org and tested without trying isos. Playiong isos is a difficult process configuring kernel modules etc and compatibilty with kernels are sketchy so it is a difficult issue to boot, so they definitely did not check.

in my opinion all the other junk should be deleted and only the packages from https://deadbeef.sourceforge.io/download.html should be used.

**Comment by Alexey-Yakovenko** [comment]
> As an example this failed horribly from source when none of the deb versions on sourceforge.net wanted to work.. 
> https://sourceforge.net/projects/deadbeef/files/debian/

This is an archive of super old releases, which should not be used anymore.
I should have deleted this folder long time ago :)

> The deb versions e.g. here at pkgs.org are absolutely junk.

I never heard about this website, and I'm not quite sure why you would want to download anything from there

> Just after I posted the issue here I found https://deadbeef.sourceforge.io/download.html.

This is the normal place where you would find the latest official downloads of deadbeef, and the files are also hosted on sourceforge.net

> All these junk packages people list needs to be removed in my opinion as they dont test with isos.

Thanks for the idea, I will consider what to do with the old files. I should definitely cleanup the folder structure in the sourceforge.net storage

**Comment by Alexey-Yakovenko** [comment]
I deleted all of the obsolete distro-specific packages.
I will move the remaining legacy files to the Archive folder.
Hope this helps.

**Comment by Alexey-Yakovenko** [comment]
> (but ignores my custom deadbeef gui layout why ?)

send me your layout, and I'll try to see why it's not working.
you can find it in the config file with "layout" in the name.

**Comment by circuitikzuser** [comment]
It is so easy to config the UI, so I recreated it very fast.

attached a zip
[1.zip](https://github.com/DeaDBeeF-Player/deadbeef/files/7993478/1.zip)

**Comment by circuitikzuser** [comment]
> I deleted all of the obsolete distro-specific packages. I will move the remaining legacy files to the Archive folder. Hope this helps.

that would help a lot and it would be nice if you keep the code aq bit closer to yourself. People go out and compile deadbeef and make debs from it just testing with wavs and normal audio files.
They NEVER test with isos...believe me and that is where this problem crept in.

Again if you need a tester to check a new version let me know.
My iso library is always setup.

**Comment by circuitikzuser** [comment]
You can change the title to whatever serves the issue the best from your perspective.
Thanks a hell of a lot for the help.

**Comment by Alexey-Yakovenko** [comment]
> People go out and compile deadbeef and make debs from it just testing with wavs and normal audio files and then people like me looking for iso functionality end up wasting a lifetime.

Isn't this the norm with opensource? You always have the choice to follow the developer recommendation, and use the tested official builds, or you can build locally, or download from a repo, and then there's no guarantee that anything would work. Developers typically can't control what people do with the source code.

**Comment by Alexey-Yakovenko** [comment]
Thanks for the config, I'll check it out

**Comment by circuitikzuser** [comment]
"Isn't this the norm with opensource?  "

Yes it is, but you are relatively invisible on sourceforge regarding your own software.
It is difficult to verify who packaged it.

pkgs.org I understand is likely not you, but I would suspect sourceforge is.

**Comment by Alexey-Yakovenko** [comment]
> Yes it is, but you are relatively invisible on sourceforge regarding your own software.

The users are not expected to go directly to sourceforge.net files section -- this is just a file storage, and it's often impossible to find which file is the one which is the latest stable release for their platform.

Instead, the users are expected to navigate to the product web site, and click Download.

**Comment by Alexey-Yakovenko** [comment]
> If the packager at the distros used https://deadbeef.sourceforge.io/download.html

Packagers never use the official downloads, they build from source, and they build differently than the official downloads.
It's not possible for the packagers to do anything else.

**Comment by circuitikzuser** [comment]
I will do that strictly in future, but it is not how I operated for the last decade and a half. It iws an unwritten rule unfortunately. Sourceforge is taken as a standard unfortunately.

**Comment by circuitikzuser** [comment]
If they build from source, do you think the will test against iso images.
I am not a beginner and come from unix and I ran in trouble to compile from source and get deadbeef to work for isos.
It is mostly gtk problems I encountered.
It is easy and straightforward to compile for non iso applications. Deadbeef works like that without effort.

From source, yes, but which sources did they use. they are human they want the job done. And if I overlooked https://deadbeef.sourceforge.io/download.html instead using the deadbeef sourceforge.net page then they and others probably did too.

**Comment by Alexey-Yakovenko** [comment]
> If they build from source, Do you think they will test against playing iso images. That is where the GTK plugin trouble happens.

You're asking the wrong person. You're basically asking me "there's this random person on the internet that compiled deadbeef and put the result up on their website. why their build of deadbeef is broken?".
I don't know -- there can be a million of reasons. They don't build the same way, they don't test the same way, and they probably don't even know what they build.

**Comment by circuitikzuser** [comment]
Exactly.

**Comment by Alexey-Yakovenko** [comment]
your layout loads just fine for me:
![image](https://user-images.githubusercontent.com/1895698/152408378-b8aa1c63-a166-4558-ae10-656faa329093.png)

**Comment by Alexey-Yakovenko** [comment]
(I was testing on current master, but should be the same with 1.9.0-beta2)
Which version did you use which didn't work?

**Comment by circuitikzuser** [comment]
Everything I tried from sourceforge.net I listed above.
In some it loaded but as soon as I play a song the interface crashed and became zombie. I could not even kill the zomby deadbeef. This is on a newly installed MX21.

The apt version that comes with MX21  was just unusable.
```
# dpkg --list |grep -i deadbeef
ii  deadbeef-static                               1.8.8-1 

```

The biggest problem now is that apt want to install deadbeef oveer the one I downloaded from your site with gdebi.
Apt thinks there is a later version than yours and the following happens. I terminated it befiore it could install.

I will have to go and pin deadbeef in the apt repositories so it doesnt install.
What a mess these distro packagers make.

```
# apt install deadbeef
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages were automatically installed and are no longer required:
  libpython2-stdlib libpython2.7-minimal libpython2.7-stdlib python2
  python2-minimal python2.7 python2.7-minimal
Use 'apt autoremove' to remove them.
The following NEW packages will be installed:
  deadbeef
0 upgraded, 1 newly installed, 0 to remove and 78 not upgraded.
Need to get 3,501 kB of archives.
After this operation, 12.6 MB of additional disk space will be used.
Get:1 http://mirrors.rit.edu/mxlinux/mx-packages/mx/repo bullseye/main amd64 deadbeef amd64 1.8.4-1~mx21 [3,501 kB]
Fetched 3,501 kB in 1s (3,870 kB/s)               
^C
```

Doing a dry-run instead
```
# apt install --dry-run deadbeef
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages were automatically installed and are no longer required:
  libpython2-stdlib libpython2.7-minimal libpython2.7-stdlib python2
  python2-minimal python2.7 python2.7-minimal
Use 'apt autoremove' to remove them.
The following NEW packages will be installed:
  deadbeef
0 upgraded, 1 newly installed, 0 to remove and 78 not upgraded.
Inst deadbeef (1.8.4-1~mx21 MX repository:21.0/mx [amd64])
Conf deadbeef (1.8.4-1~mx21 MX repository:21.0/mx [amd64])

```
So clearly the gdebi install from your site which now works perfectly was not registered as installed  by apt or dpkg. It still sees deadbeef (1.8.4-1~mx21)
I dont know if that can be switched off while creating a deb, sort of a stealth install. That is the only thing that could be the matter on your side.

These issues are not really your problem but rather a sloppy packager.
I will just pin MX and never allow apt to install deadbeef, then in future only get your debs at sourceforge.io.

**Comment by circuitikzuser** [comment]
> your layout loads just fine for me: ![image](https://user-images.githubusercontent.com/1895698/152408378-b8aa1c63-a166-4558-ae10-656faa329093.png)

Just for the record did you test it with a virtual cdrom  as created by cdemu ??

**Comment by Alexey-Yakovenko** [comment]
> Everything I tried from sourceforge.net I listed above.

that's the funny thing -- you didn't :) You didn't list a single thing.

> Just for the record did you test it with a virtual cdrom as created by cdemu ??

didn't you say this is not a problem anymore, and the only problem is the custom layout not loading?

**Comment by circuitikzuser** [comment]
Everything on the page I listed I tried ! Why copy it all over ?

No, the files I sent you did not work.
I had to recreate the entire interface again, from scratch and now it works.

**Comment by circuitikzuser** [comment]
I just wanted to get a copy of the deb from your site as it works perfectly, but i saw the site was down.
In case you do not know.

https://deadbeef.sourceforge.io/download.htm

An error has been encountered in accessing this page.

1. Server: deadbeef.sourceforge.io
2. URL path: /download.htm
3. Error notes: NONE
4. Error type: 404
5. Request method: GET
6. Request query string: NONE
7. Time: 2022-02-04 09:17:14 UTC (1643966234)

Reporting this problem: The problem you have encountered is with a project web site hosted by SourceForge.net. This issue should be reported to the SourceForge.net-hosted project (not to SourceForge.net).

If this is a severe or recurring/persistent problem, please do one of the following, and provide the error text (numbered 1 through 7, above):

    Contact the project via their [designated support resources](https://sourceforge.net/support/prweb-lookup.php?host=deadbeef.sourceforge.io&support=1).
    Contact the project administrators of this project via email (see the upper right-hand corner of the [Project Summary page](https://sourceforge.net/support/prweb-lookup.php?host=deadbeef.sourceforge.io) for their usernames) at user-name@users.sourceforge.net

If you are a maintainer of this web content, please refer to the [Site Documentation regarding web services](https://sourceforge.net/p/forge/documentation/Project%20Web%20Services/) for further assistance.

NOTE: As of 2008-10-23 directory index display has been disabled by default. This option may be re-enabled by the project by placing a file with the name ".htaccess" with this line:

**Comment by Alexey-Yakovenko** [comment]
> You keep misunderstanding me about this issue.

yes, indeed -- I didn't understand the issue, because you wrote that the layout doesn't load. Obviously, I tested loading the layout - and it worked.

loading layout has nothing to do with cd images, so I couldn't make any connection.

I'll try to configure CDEmu in a virtual machine, and test with it.

> https://deadbeef.sourceforge.io/download.htm

There's no such page. It's download.html, not htm

**Comment by Alexey-Yakovenko** [comment]
> Everything on the page I listed I tried ! Why copy it all over ?

Whatever works for you. I don't understand what you mean when you say "I listed", because to me the only thing you listed if that you downloaded something from https://deadbeef.sourceforge.io/download.html
Is that what you meant?

**Comment by circuitikzuser** [comment]
Sorry I had a broken link. I will delete my post above.

**Comment by Alexey-Yakovenko** [comment]
> I dont know if that can be switched off while creating a deb, sort of a stealth install. That is the only thing that could be the matter on your side.

TBH I don't even understand what you mean here :)

> Your deb on sourceforge.io works perfectly so the problem is solved.
> The souceforge.NET versions does not work with virtual cds.

Can you provide concrete links to the version which doesn't work, and to the version which works?
I'm confused about this, because there are no files on sourceforge.io -- only on sourceforge.net.

**Comment by Alexey-Yakovenko** [comment]
If you don't know how to get the links -- you can right click on the link, and copy it to clipboard, in Chrome the menu item is called "Copy Link Address". Then you can paste both links here on github.

**Comment by Alexey-Yakovenko** [comment]
I installed cdemu in a VM, and ran deadbeef with your layout, then played the audio CD image without any problems.
So, I'm still quite in doubt that I understand correctly what kind of problems are you experiencing.

**Comment by circuitikzuser** [comment]
I'll make a video then. So you can see.

**Comment by Alexey-Yakovenko** [comment]
Please also remember to include the URL of the file which you installed / used.

**Comment by circuitikzuser** [comment]
> I installed cdemu in a VM, and ran deadbeef with your layout, then played the audio CD image without any problems. So, I'm still quite in doubt that I understand correctly what kind of problems are you experiencing.

I have no clue what you are talking about. I know how to copy links.

**Comment by circuitikzuser** [comment]
> If you don't know how to get the links -- you can right click on the link, and copy it to clipboard, in Chrome the menu item is called "Copy Link Address". Then you can paste both links here on github.

I really dont know what you talk about. I dont have a problem copying links.

**Comment by Alexey-Yakovenko** [comment]
> I have no clue what you are talking about. I know how to copy links.

Cool, then I'll wait until you post them :)

**Comment by circuitikzuser** [comment]
Last thing before I make a video about the main issue. 
Since I installed the deb from your site, which works nicely with isos,  my playlists however  dont play with the usual order they are in a list.
It jumps around in my list and do not play from top to bottom.
Also, the sorting options at the top  tabs are missing text.

see image here.
`https://i.imgur.com/PcOgkBe.pnghttps://i.imgur.com/PcOgkBe.png` 
Again
What is shown is my preferred ordering, but now it is played by the numbers in the first column, so it jumps around the list when playing.
Secondly the ordering titles/text in tabs are missing.

**Comment by Alexey-Yakovenko** [comment]
@circuitikzuser I can't help you with most of your problems, until I find out which file you have downloaded and installed, which operating system you're running, which GTK version / GTKUI plugin (2 or 3), and maybe I'll need some extra details.

> It jumps around in my list and do not play from top to bottom.

Did you check that the shuffle is turned off?

> Secondly the ordering titles/text in tabs are missing.

The weird colors bug that you're seeing was fixed (I think) quite recently, and you're welcome to validate the fix by running [deadbeef 1.9.0-beta3](https://sourceforge.net/projects/deadbeef/files/travis/linux/1.9/deadbeef-static_1.9.0-beta3-1_x86_64.tar.bz2/download)

**Comment by Alexey-Yakovenko** [comment]
These are the colors I'm seeing, when running deadbeef with your config file.
Do they look correct?
![image](https://user-images.githubusercontent.com/1895698/152603884-8f20fda9-30a4-459f-b1c6-926efe39aa81.png)

**Comment by Alexey-Yakovenko** [comment]
So far, I couldn't confirm / reproduce a single problem that you posted.

**Comment by circuitikzuser** [comment]
Ok then .
I must be making it all up then and waste my own time for hours doing so.

**Comment by Alexey-Yakovenko** [comment]
Well, you're probably not making it up, but it could have went better if you were more forward-coming with providing the requested information, and try to solve one problem at a time.

**Comment by Alexey-Yakovenko** [comment]
Since you seem to be frustrated to the point of not wanting to communicate anymore,
I'll give you some feedback. Hope you don't mind:

* When originally posting the bug report, you should have started by filling out the template, instead of deleting it and writing freeform. In this case, your bug report would not have been closed as invalid from the very start.
* When I ask for the information about your software setup -- it means that I need it to help you, and most likely I would not be able to help if you don't provide the requested information. Which is what happened
* It is best to focus on solving one problem at a time. If you have multiple unrelated problems -- it's best to post them as separate bugs. This way the conversation would go in a more structured and more constructive way.

**Comment by Alexey-Yakovenko** [comment]
> "https://deadbeef.sourceforge.io/download.html"
> Posted that right in the first few posts.

you seriously think that you have installed an html?

**Comment by circuitikzuser** [comment]
Here are my colors.
Again using your latest deb on a clean MX21 installation.
Clearly not the same what you posted.
[https://imgur.com/LiEeBAT.png](url)

**Comment by Alexey-Yakovenko** [comment]
I'm still waiting for the information which particular file you have installed :)
html page url is not what I need.
I need to know the exact URL which you clicked ON that page.
> I am using the Latest MX21 Distro.

**Comment by Alexey-Yakovenko** [comment]
> Clearly not the same what you posted.

yeah, because you didn't open the same preferences tab.
Have you tried the 1.9.0-beta3 build which I linked? Does it solve the green text problem?

**Comment by circuitikzuser** [comment]
So exactly which preferences tab do I need to know.
Usually a program has only a single set of preferences.

I am using deadbeef-static_1.8.8-1_amd64.deb

> > "https://deadbeef.sourceforge.io/download.html"
> > Posted that right in the first few posts.
> 
> you seriously think that you have installed an html?

What is your problem ? 
deadbeef-static_1.8.8-1_amd64.deb

**Comment by Alexey-Yakovenko** [comment]
> I am now using your latest deb from here.

It seems like you really have a problem with copy&pasting urls on the internet, so I'll make a guess that you are using this file at the moment:

https://sourceforge.net/projects/deadbeef/files/travis/linux/1.8.8/deadbeef-static_1.8.8-1_amd64.deb/download

I will also assume that when using this file you don't encounter any problems, except the green color in the playlist tab titles.

This problem should have been fixed in the 1.9.0 (which I linked).
This is all of the help I could provide, and on this note I have to wrap up.

Good bye.
